matter_server/common/const.py

```diff
@@ -0,0 +1,12 @@
+"""Constants that are shared between server and client."""
+
+# schema version is used to determine compatibility between server and client
+SCHEMA_VERSION = 1
+
+# we allow the schema version to be one version behind
+MIN_SCHEMA_VERSION = SCHEMA_VERSION - 1
```

Comment:
I don't think that we have to make this dynamic, we could maybe allow 3 versions if they are minor extensions?

Reply:
So basically, we just support as much as we can. Sometimes method signatures change a bit over time but it's not that much.